data_provenance/to-dolma.py

```diff
@@ -132,11 +140,12 @@ def file_to_dolma(path: str, include_df: str, source_name: str = SOURCE_NAME):
                     "license_url": license_urls,
                     "language": langs,
                     "url": url,
-                    "dataset_id": ex["dataset"],
+                    "dataset_id": dataset_id,
                     "response": target_text,
                 },
             }
         )
+
```

Comment:
Great! Also collect `seen_dataset_ids` then check here:

```
unseen_ids = valid_ids - set(seen_dataset_ids)
print(f"Unseen Datasets: {len(unseen_ids)} | {unseen_ids}")
```

Reply:
Will do

Reply:
Or maybe if no assert used:

```python
if dataset_id in valid_ids:
  seen_dataset_ids.add(dataset_id)
else:
  logger.warning(f"Dataset ID '{dataset_id}' not found in include.csv")
```

Reply:
@conceptofmind wait, this looks like a bug of some sort. There are 273 datasets (including many simple ones like dolly, helpsteer,  and many fc-flan/sni ones that are all omitted). I thought the new HF object had all of these?

Reply:
Looks like it is just a logging issue. Will be moved outside the loop

Reply:
LGTM! Thanks for investigatin!

Reply:
Should be resolved. Can reopen if not